pkdiagram/scene/event.py

```
@@ -397,31 +362,81 @@ def removeDynamicProperty(self, attr):
     def clearDynamicProperties(self):
         self.dynamicProperties = []
 
-    # Variables
+    # Person
+
+    def person(self) -> "Person":
```

Ruff (0.13.3)

Undefined name `Person`

(F821)